extension/wrappers.h

```diff
@@ -254,19 +264,61 @@ class CFrameSnapshotManager
 class CBaseEntity :
 	public IServerEntity
 {
+public:
+	static inline uint16_t m_iTeamNumOffset;
```

Comment:
If you've decided to use `static` here, thats probably fine.
However, should always initialize your variables.